### ISSUE TITLE
Enable "focus" on tabs component

### ISSUE THREAD

**Issue by faihegberg** [covid-projections/act-now-packages#298]
Currently when we focus on a tab without selecting it, there's no visible indication that we're focusing on that tab. This can be confusing for users navigating the tabs without a mouse, since there's no way to know which tab they're on.

The tab receiving focus should have a clear and visible indication that it is currently receiving focus.
